.github/workflows/run-tests-and-checks.yml

```diff
@@ -0,0 +1,39 @@
+name: Run tests and checks
+
+on:
+  push:
+
+jobs:
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['3.1', '3.0', '2.7']
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+      - name: Install rubocop
+        run: gem install rubocop
```

Comment:
shouldn't we do `bundle install` ?

Reply:
I originally wanted to avoid running another `bundle install` but then discovered that we can cache it and it doesn't take too long anyway. And also it will help us keep consistency with versions, so I'll make the change 😄

Reply:
done